files/k8s-postdeploy

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+function aws_setup {
+    integrator=$(juju status --format=json | jq '.applications["aws-integrator"]' -r)
+    if [ $integrator = "null" ]; then
+        echo 'deploying aws-integrator'
+        juju deploy cs:~containers/aws-integrator
+    else
+        echo 'aws-integrator already deployed'
+    fi
+    juju add-relation aws-integrator kubernetes-master
+    juju add-relation aws-integrator kubernetes-worker
```

Comment:
I guess the second relation is to another app?

Reply:
Yeah, kubernetes in both core and canonical distribution is deployed w/ two applications, -worker and -master. Same charm, but different app deployments.